content/projekte/2030-watch.en.md

```diff
@@ -17,7 +17,7 @@ more_text: |-
 2030Watch monitors the implementation of the Sustainable Development Goals (SDGs) in Germany.
 The SDGs were set by the United Nations in September 2015 and will be implemented by 2030.
 All members of the United Nations have committed to realizing these 17 goals and 169 targets.
-The goals include the eradication of poverty and inequality between and within countries as well as action against climate change.
+The goals include the eradication of poverty and inequality between and within countries, as well as action against climate change.
 2030Watch primarily uses data from the OECD, the World Bank and other international organizations.
 The project will also propose indicators for goals which are hard to operationalize, such as fair trade relations.
 The launch of the 2030 Watch website is planned for the end of October 2015.
```

Comment:
There's a budget here, but not projects are publishing budgets. What makes most sense?

Reply:
A few things to note here: 
- budgets can be expressed in several different ways (per year, total budget, fixed costs, personal costs etc.) and need to be updated correspondingly  
- the yearly report contains budget figures for all projects and should be the main reference point for a more detailed view of a given projects' budget
- the project view should therefore not contain budget figures but only information on the funding entity
- some projects don't have a budget (ehrenamtlich), this could be expressed more visibly